install/tools/ipa-replica-conncheck

```diff
@@ -346,9 +346,9 @@ class PortResponder(threading.Thread):
                 sock.listen(1)
 
             logger.debug('%d %s: Started listening', port, proto)
-        except socket.error as e:
+        except socket.error:
             logger.warning('%d %s: Failed to bind', port, proto)
-            logger.debug("%s", traceback.format_exc(e))
+            logger.debug("%s", traceback.format_exc())
```

Comment:
Hi standa,

I think we could use just `logger.debug(traceback.format_exc())` here. AFAIK there is no need to use `"%s"` in this case.
However, in the entire code base we're using `logger.debug("%s", traceback.format_exc())`, so even though I prefer the first option, it's up to you; just let me know.